validation_tool/schemas/facilities_a15_schema.py

```diff
@@ -0,0 +1,357 @@
+from pandas import Timestamp
```

Comment:
same as above. what schema is that, what does it map to

Reply:
I deleted this as I pivoted away from the A-15 for now to higher priority things. It was an auto-generated script that can easily be recreated.